e2etest/testcases.json

```diff
@@ -47,6 +47,10 @@
 		"case_name": "otlp_trace_resourcedetection_ecs",
 		"platforms": ["ECS"]
 	},
+	{
+		"case_name": "containerinsight_ecs_prometheus",
+		"platforms": ["ECS"]
```

Comment:
Please ensure this change is also made in the `testcases.json` file in the test-framework